CSharp/2-doubly.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LinkedList
+{
+    class DoubleLinkedList<T>
+    {
+        private Node last = null;
+
+        private class Node
+        {
+            internal T item;
+            internal Node next;
+            internal Node previous;

```

Comment:
Modifier can be **public** because class `Node` is inner private, so fields can be accessed only in `DoubleLinkedList` class scope.